CollAction/Frontend/app/global/RegisterReferer.ts

```diff
@@ -0,0 +1,29 @@
+﻿function parseUtmParameters() : string {
+    const utmKeys = ["utm_source", "utm_medium", "utm_campaign", "utm_term", "utm_content"];
+    const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Whatch out; URLSearchParams isn't supported in IE11.

Reply:
You can either use a polyfill or your own function like https://gomakethings.com/getting-all-query-string-values-from-a-url-with-vanilla-js/

Reply:
I added a polyfill